CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ directly impact users rather than highlighting other crucial architectural updat
 ### Changed
 - The way how the balances (zatoshi amounts/values) are represented has been updated accoridng to the latest requirements. In general any zatoshi value has 2 major states, expanded or abbreviated. Trailing zeroes are trimmed when expanded.
 - The `Balances` screen has been redesigned: new progress bar with the status of the synchronization, all balances available. Penging fields are disabled for now and show only zeroes until support from the SDK is implemented.
+- When the send button is tapped, the sending title + progress is showed instead of just the progress. Also when the send is done and redirect to the Account page is done, the sending transaction is already populated in the list. The lag between it was presented has been fixed.
```

Comment:
Proposing these minor changes: `showed` -> `shown`. `Also when the send is done and redirect to the Account page is done` -> `Also, when the send is done, and the redirect to the Account page is done`.

Reply:
Thx, incorporated suggestions + I changed the word `progress` to a `spinner` so it's more clear what component I mean.